dlp/api/Snippets/SendDataToTheHybridJobTrigger.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_inspect_send_data_to_hybrid_job_trigger]
+
+using System;
+using System.Threading;
+using Google.Cloud.Dlp.V2;
+
+public class SendDataToTheHybridJobTrigger
+{
+    public static DlpJob SendToHybridJobTrigger(
+       string projectId,
+       string jobId,
+       string jobPath,
+       string text = null)
+    {
+        // Instantiate the dlp client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct the hybrid finding details which will be used as metadata with the content.
+        // Refer to this for more information: https://cloud.google.com/dlp/docs/reference/rest/v2/Container
+        var findingDetails = new HybridFindingDetails
+        {
+            ContainerDetails = new Container
+            {
+                FullPath = "10.0.0.2:logs1:aap1",
+                RelativePath = "app1",
+                RootPath = "10.0.0.2:logs1",
+                Type = "System Logs"
+            }
+        };
+
+        // Construct the hybrid content item using the finding details and text to be inspected.
+        var hybridContentItem = new HybridContentItem
+        {
+            Item = new ContentItem { Value = text ?? "My email is ariel@example.org and name is Ariel." },
+            FindingDetails = findingDetails
+        };
+
+        // Activate the Job Trigger.
+        var activate = new ActivateJobTriggerRequest
+        {
+            JobTriggerName = new JobTriggerName(projectId, jobId),
+            Name = jobPath
+        };
+
+        // Call the API to activate the trigger.
+        DlpJob triggerJob = dlp.ActivateJobTrigger(activate);
+
+        // Construct the request using hybrid content item.
+        var request = new HybridInspectJobTriggerRequest
+        {
+            HybridItem = hybridContentItem,
+            JobTriggerName = new JobTriggerName(projectId, jobId),
+            Name = jobPath,
+        };
+
+        // Call the API.
+        HybridInspectResponse response = dlp.HybridInspectJobTrigger(request);
+
+        // Check the activated job status to fetch the inspect details.
+        do
+        {
+            triggerJob = dlp.GetDlpJob(new GetDlpJobRequest
+            {
+                Name = triggerJob.Name
+            });
+            Thread.Sleep(5000);
+        } while (triggerJob.InspectDetails.Result.ProcessedBytes <= 0);
```

Comment:
Can this really be less than zero? If not, it's better to only check for it being zero to avoind confusing users.

Reply:
Changed the condition. Thanks.